File: examples/csharp/BulkGeolocation.cs
// Runnable example: Bulk IP Geolocation (POST /v1.0/geolocation)
// Parameters for bulkGeolocation (POST /v1.0/geolocation):
//   - body: BulkGeolocationRequest (required) -- request body object
using System;
using WhoisFreaks.Api;
using WhoisFreaks.Client;
using WhoisFreaks.Model;

class BulkGeolocation {
    static void Main() {
        var config = new Configuration { BasePath = "https://api.whoisfreaks.com" };
        config.AddApiKey("ApiKeyAuth", "YOUR_API_KEY");  // set once
        var api = new GeolocationApi(config);
        var result = api.BulkGeolocation(new BulkGeolocationRequest());
        Console.WriteLine(result);
    }
}

File: examples/java/BulkGeolocation.java
// Runnable example: Bulk IP Geolocation (POST /v1.0/geolocation)
// Parameters for bulkGeolocation (POST /v1.0/geolocation):
//   - apiKey (string, required): Your WHOISFreaks API key
//   - body: BulkGeolocationRequest (required) -- request body object
import com.whoisfreaks.client.ApiClient;
import com.whoisfreaks.client.Configuration;
import com.whoisfreaks.client.api.GeolocationApi;
import com.whoisfreaks.client.model.BulkGeolocationRequest;

public class BulkGeolocation {
    public static void main(String[] args) throws Exception {
        ApiClient client = Configuration.getDefaultApiClient();
        client.setBasePath("https://api.whoisfreaks.com");
        GeolocationApi api = new GeolocationApi(client);
        var resp = api.bulkGeolocationWithHttpInfo("YOUR_API_KEY", new BulkGeolocationRequest());
        System.out.println("status: " + resp.getStatusCode());
        System.out.println(resp.getData());
    }
}
